go.mod

```diff
@@ -1,6 +1,6 @@
 module kubevirt.io/containerized-data-importer
 
-go 1.17
+go 1.18
```

Comment:
Should probably update the go.mod in staging well

Reply:
Yup done